Vagrantfile

```diff
@@ -33,13 +42,14 @@ Vagrant.configure("2") do |config|
 
     config.vm.define "centos" do |b|
         b.vm.box = "centos/7"
+        b.vm.provision "Fix shared folder permissions", type: "shell", inline: "chown -R vagrant /vagrant"
         b.vm.provision "Install dependencies", type: "shell", inline: centos_install
         b.vm.provision "Build binary", type: "shell", inline: build, run: "always"
     end
 
     config.vm.define "darwin" do |b|
-        b.vm.box = "jhcook/osx-elcapitan-10.11"
+        b.vm.box = "jhcook/macos-sierra"
```

Comment:
Please revert this. For building the binaries, older is better, as it ensures that we build a binary that can run on both El Capitan and Sierra.